tests/system/test_compiler.py

```diff
@@ -129,9 +136,8 @@ def test_window_function(alltypes, project_id, dataset_id):
     assert result == expected
 
 
+@pytest.mark.skipif(IBIS_VERSION <= IBIS_1_VERSION, reason="requires ibis 2.x")
```

Comment:
Nit: I think we can remove this, as we dropped ibis 1.0 support in this package already.